CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# dbt_jira v0.12.0
+## 🚨 Breaking Changes 🚨:
+- Fixed `jira__daily_issue_field_history` model to make sure component values are correctly joined into our issue models ([#81](https://github.com/fivetran/dbt_jira/pull/81)).
+
```

Comment:
```suggestion
- Please note, a `dbt run --full-refresh` will be required after upgrading to this version in order to capture the updates.
```
Maybe add something like this? (took this from the v0.9.0 release for refefence.)